CHANGELOG.md

```diff
@@ -42,9 +46,15 @@
 
 ### Internal
 
+- Various build improvements and minor refactoring captured during the effort to unify
```

Comment:
Where does this come from? Merge issue?

Reply:
Yes. GitHub complained about a bad merge commit, and I attempted to resolve it. I'm not sure exactly where it belongs, so I made a best guess.

Reply:
Heh, and the changelog.md is out of date again. I think I'll let the release manager handle that. :wink:

Reply:
Ok no worries! I'll take a look!

Reply:
Done

Reply:
Thank you!